build.gradle

```diff
@@ -69,7 +69,7 @@ group = 'com.github.broadinstitute'
 
```

Comment:
Can you update the `java -jar picard*-all.jar` line in the README to use `picard.jar` instead, and also mention that a jar with the full version in its name is also emitted.